apps/namadillo/src/atoms/ledger/atoms.ts

```diff
@@ -0,0 +1,75 @@
+import { ledgerUSBList } from "@namada/sdk/web";
+import { LedgerError } from "@zondax/ledger-namada";
+import { isLedgerAccountAtom } from "atoms/accounts";
+import { atom } from "jotai";
+
+import { atomWithQuery } from "jotai-tanstack-query";
+import { getSdkInstance } from "utils/sdk";
+
+export type LedgerStatus = {
+  connected: boolean;
+  errorMessage: string;
+};
+
+const ledgerStatusStopAtom = atom(false);
+
+export const ledgerStatusAtom = atomWithQuery<LedgerStatus | undefined>(() => {
+  return {
+    refetchInterval: 1000,
+    queryKey: ["ledger-status"],
+    queryFn: async () => {
+      const devices = await ledgerUSBList();
+
+      if (devices.length > 0) {
+        try {
+          // Disable console.warn to prevent the warning from showing up in the UI in the loop
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (console as any).warnOld = console.warn;
+          console.warn = () => {};
```

Comment:
I see it has the condition `isLedgerAccount` before doing this. But I'm felling it could create border cases in the future

What do you think if we only skip the undesired message with a filter? Or even just accept the warn message?

If filter, something like this

```ts
console.warn = (...args) => {
   if (String(args[0]).startsWith("something")){
     return;
   }
   console.warnOld(...args)
};
```

Reply:
@mateuszjasiuk pinging you since you'll have a better idea what is going on here